### PR TITLE
Hw 26

### PR DESCRIPTION
**Pull request by MrNopi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,23 +1,13 @@
...
 
-# BlueJ files
-*.ctxt
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
 # Package Files #
 *.jar
 *.war
```
Comment (Tarabarnik) [inline .gitignore]: they appear only at target/, so redundant here

File: .gitignore [hunk 2]
```diff
@@ -1,23 +1,13 @@
+target/
+.idea/
+*.iml
+
 # Compiled class file
 *.class
```
Comment (Tarabarnik) [inline .gitignore]: same

File: .idea/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```
Comment (Tarabarnik) [inline .idea/.gitignore:2]: files from .idea should not be pushed. plese, fix in the next pr

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                        <source>${jdk.version}</source>
+                        <target>${jdk.version}</target>
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+
+</project>
```
Comment (Tarabarnik) [inline pom.xml]: add empty line

File: src/main/java/mate/academy/util/HibernateUtil.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * Created by yusufcakmak on 8/3/15.
+ */
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+
+    //XML based configuration
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/util/HibernateUtil.java:8]: remove comment please

File: src/main/java/mate/academy/util/HibernateUtil.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new RuntimeException("Error creating session factory", e);
+        }
+    }
+    public static SessionFactory getSessionFactory() {
+        return sessionFactory;
+    }
+}
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/util/HibernateUtil.java]: add empty line

File: src/main/resources/hibernate.cfg.xml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        <property name="connection.password">12345</property>
+        <property name="dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class = "mate.academy.model.Movie"/>
+    </session-factory>
+
+
+</hibernate-configuration>
```
Comment (Tarabarnik) [inline src/main/resources/hibernate.cfg.xml]: add empty line

File: .gitignore (hunk 1 above)
Comment (MrNopi) [reply .gitignore]: done

File: .gitignore (hunk 2 above)
Comment (MrNopi) [reply .gitignore]: done

File: .idea/.gitignore (hunk 3 above)
Comment (MrNopi) [reply .idea/.gitignore:2]: roger that

File: pom.xml (hunk 4 above)
Comment (MrNopi) [reply pom.xml]: done

File: src/main/java/mate/academy/util/HibernateUtil.java (hunk 5 above)
Comment (MrNopi) [reply src/main/java/mate/academy/util/HibernateUtil.java:8]: done

File: src/main/java/mate/academy/util/HibernateUtil.java (hunk 6 above)
Comment (MrNopi) [reply src/main/java/mate/academy/util/HibernateUtil.java]: done

File: src/main/resources/hibernate.cfg.xml (hunk 7 above)
Comment (MrNopi) [reply src/main/resources/hibernate.cfg.xml]: done

**Review by Tarabarnik — APPROVED** [review]

**Pull request merged**